modules/install/mod_settings/config.xml

```diff
@@ -174,6 +174,25 @@
       </entry>
     </entries>
   </typedefinition>
+  <typedefinition>
+    <head>
+      <name>passwordhashalgo</name>
```

Comment:
I am more in favor of
- splitting the long configuration parameter a bit for readability like `password_hash_algo`
- not using abbreviations and making it explicit, because we are not limited in space here like `password_hashing_algorithm`

I can take this on to change it.